metadata-ingestion/src/datahub/ingestion/source/nifi.py

```diff
@@ -1001,8 +1035,6 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
             self.report.report_failure(self.config.site_url, "Failed to authenticate")
             return
 
-        self.session.headers.update({"Content-Type": "application/json"})
```

Comment:
this is not required. Most of the nifi requests connector calls accept content-type: */*